nixtract/find-attribute-paths.nix

```
@@ -16,11 +17,16 @@ let
 
   # Arguments have to be taken from environment when using `nix` command
   targetFlakeRef = builtins.getEnv "TARGET_FLAKE_REF";
+  targetAttribute = builtins.getEnv "TARGET_ATTRIBUTE_PATH";
```

_beware my nitpicking_ :smile: 

```suggestion
  targetAttributePath = builtins.getEnv "TARGET_ATTRIBUTE_PATH";
```